pkg/tlsx/jarm/jarm.go

```diff
@@ -12,12 +12,19 @@ import (
 	"github.com/projectdiscovery/tlsx/pkg/connpool"
 )
 
+const poolCount = 3
+
 // fingerprint probes a single host/port
-func HashWithDialer(dialer *fastdialer.Dialer, host string, port int, timeout time.Duration) (string, error) {
+func HashWithDialer(dialer *fastdialer.Dialer, host string, port int, duration int) (string, error) {
```

Comment:
Wouldn't the timeout of type `time.Duration` be more generic?

Reply:
yes, but since it is used during calculation of context timeout as well, time.Duration is not ideal as multiplying it by poolCount leads to wrong time